Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "sk8brd"
 version = "0.1.0"
 edition = "2021"
 authors = ["Konrad Dybcio <konradybcio@kernel.org>"]
+license = "GPL-2.0"
```

Comment:
I think most of Rust crates use more permissive licence. Would you consider following the stream?

Reply:
LGPL is the most I can do at the moment, may reconsider in the future..

Reply:
Then you'd have to ask all contributors to relicense. All our tools were using BSD-3-Clause I believe. Why don't we follow the custom?

Reply:
Fine, I'll cave in this time, mostly because I'm also reviewing cdba code in parallel and that's BSD3..